### ISSUE TITLE
🐛 When a ScriptTask fails, the LET stucks at the failed script task

### ISSUE THREAD

**Issue by robpal94** [process-engine/bpmn-studio#1312]
#### Describe your issue

When a Faulty script task is executed (for example, if the definition contains a Syntax Error), the let is stucks on the Script Task. 

Also the Process Instance which contains the faulty ScriptTask will be displayed in the Dashboards TaskList.

The underlying ProcessEngine produces the following output: 

```
Executing ScriptTask instance 44eda173-814c-4a8a-9e17-acfb2c9c5453
2019-01-24T14:33:13.165Z - error: [processengine:runtime:startup] -- An unhandled exception was caught! Error: --
2019-01-24T14:33:13.165Z - error: [processengine:runtime:startup] SyntaxError: Unexpected identifier
2019-01-24T14:33:13.165Z - error: [processengine:runtime:startup] -- end of unhandled exception stack trace --
```

#### How can others reproduce the issue?

Try to execute a Process which contains a Faulty Script Task ´

#### Possible solution

The Process should be aborted and the LET should react to that. Also, I wont expect the ProcessInstance with the faulty ScriptTask to show up in the TaskList.

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

**Comment by robpal94** [comment]
@ElRaptorus Do you know how this case is handled on the ProcessEngine side? I know that we are able to catch faulty ScriptTasks, but what about the ProcessInstance itself? Is the Process with the faulty ScriptTask aborted properly or will it also be stuck at this _Error_ State?

**Comment by ElRaptorus** [comment]
@robpal94 When such an error occurs, the FlowNodeInstance's state will be changed to `error`. 
At that point, it should not logner register as active.

**Comment by Vyperus** [comment]
So I just tested this.
Since no diagram was provided I had to guess what a "faulty" script task is in this case.
Here is my diagram:

[ScriptTaskTest.bpmn.zip](https://github.com/process-engine/bpmn-studio/files/2878845/ScriptTaskTest.bpmn.zip)

With the current Release auf BPMN-Studio (4.21.0) and the current PE (6.0.0) i wasn't able to reproduce this Issue. 
Correct me if I am wrong.

Closing this for now.
